packages/blockfrost/src/blockfrostWalletProvider.ts

```diff
@@ -306,12 +248,38 @@ export const blockfrostProvider = (options: Options, logger = dummyLogger): Wall
     ];
   };
 
+  const fetchJsonMetadata = async (txHash: Cardano.Hash16): Promise<Cardano.MetadatumMap | null> => {
+    try {
+      const response = await blockfrost.txsMetadata(txHash);
+      return response.reduce((map, metadatum) => {
+        // Not sure if types are correct, missing 'label', but it's present in docs
```

Comment:
It was fixed in https://github.com/blockfrost/blockfrost-js/releases/tag/v2.0.1. [2.0.0]( https://github.com/blockfrost/blockfrost-js/releases/tag/v2.0.0) appears to only have one breaking change, so the upgrade should be routine.